src/components/standalone/backup_and_restore/BackupContent.vue

```diff
@@ -0,0 +1,284 @@
+import ModalRunBackup from '@/components/standalone/backup_and_restore/ModalRunBackup.vue'
+import DrawerSetPassphrase from '@/components/standalone/backup_and_restore/DrawerSetPassphrase.vue'
+
+const { t } = useI18n()
+
+let loading = ref(true)
+let loadingPassphrase = ref(true)
+let isEnterprise = ref(false)
+let isSetPassphrase = ref(false)
+let showDownloadModal = ref(false)
+let showPassphraseDrawer = ref(false)
+let showRunBackupModal = ref(false)
+let unitName = ref(undefined)
+let seletedBackup = ref('')
+let listBackups: any = ref([])
+
+let objNotification = {
+  notificationTitle: '',
+  notificationDescription: ''
+}
+
+let error = ref(false)
+let errorSubscription = ref({ ...objNotification })
+let errorHostname = ref({ ...objNotification })
+let errorGetBackup = ref({ ...objNotification })
+let errorIsPassphrase = ref({ ...objNotification })
+let successNotificationRunBackup = ref({ ...objNotification })
+
+onMounted(() => {
+  getSubscription()
+  getHostname()
+  getIsPassphrase()
+})
+
+async function getSubscription() {
+  try {
+    let res = await ubusCall('ns.subscription', 'info', {})
+    if (res?.data?.systemd_id && res?.data?.active) {
+      isEnterprise.value = true
+      await getBackups()
+    }
+  } catch (exception: any) {
+    error.value = true
+    errorSubscription.value.notificationTitle = t('error.cannot_retrieve_subscription_info')
+    errorSubscription.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  } finally {
+    loading.value = false
+  }
+}
+
+async function getHostname() {
+  try {
+    let systemInfo = await ubusCall('system', 'board')
+    unitName.value = systemInfo.data.hostname
+  } catch (exception: any) {
+    error.value = true
+    errorHostname.value.notificationTitle = t('error.cannot_retrieve_system_board')
+    errorHostname.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  }
+}
+
+async function getIsPassphrase() {
+  isSetPassphrase.value = false
+  try {
+    let res = await ubusCall('ns.backup', 'is-passphrase-set', {})
+    if (res?.data?.values?.set) {
+      isSetPassphrase.value = true
+    }
+  } catch (exception: any) {
+    error.value = true
+    errorIsPassphrase.value.notificationTitle = t('error.cannot_retrieve_passphrase')
+    errorIsPassphrase.value.notificationDescription = t(getAxiosErrorMessage(exception))
+  } finally {
+    loadingPassphrase.value = false
+  }
+}
+
+async function getBackups() {
+  if (isEnterprise.value) {
+    try {
+      let res = await ubusCall('ns.backup', 'registered-list-backups')
+      if (res?.data?.values?.length) listBackups.value = res.data.values
+    } catch (exception: any) {
+      errorGetBackup.value.notificationTitle = t('error.cannot_retrieve_backup')
+      errorGetBackup.value.notificationDescription = t(getAxiosErrorMessage(exception))
+    }
+  }
+}
+
+function oepenDownloadEnterprise(file: string) {
+  showDownloadModal.value = true
+  seletedBackup.value = file
+}
```

Comment:
You can remove this entirely, handle the show of the modal if `seletedBackup` is undefined or not.